install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,118 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
+rebalancing as cluster members restart.
+
+We present two restart procedures: rolling-restart and full-restart. Where a rolling
+restart gracefully applies appropriate changes to the Elasticsearch cluster without
+down time (provided 3 masters are configured) and a full restart safely applies
+major changes without risk to existing data.
+
+[[elasticsearch-rolling-restart]]
+=== Performing an Elasticsearch rolling cluster restart
+
+A rolling restart is appropriate, and recommended, when any of the following changes are made:
+
+* nodes on which Elasticsearch pods run require a reboot
+* logging-elasticsearch configmap
+* logging-es-* deployment configuration
+* new image deployment, or upgrade
+
+This will be the recommended restart policy going forward.
+
+[NOTE]
+====
+Any action you do for a ES cluster will need to be repeated for the ops cluster
+if `openshift_logging_use_ops` was configured to be `True`.
+====
+
+1. Prevent shard balancing happening when we are bring down nodes on purpose
```

Comment:
Prevent shard balancing when purposely bringing down nodes from any ES pod in the cluster:

Reply:
The "From any ES pod in the cluster:" is more a note on how to execute the command than why we are running the command.